awx/ui_next/src/screens/Template/shared/InventoriesLookup.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react';
+import { string, func, bool } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { FormGroup, Tooltip } from '@patternfly/react-core';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+
+import { InventoriesAPI } from '@api';
+import Lookup from '@components/Lookup';
+import { Inventory } from '../../../types';
+
+const getInventories = async params => InventoriesAPI.read(params);
+
+class InventoriesLookup extends React.Component {
+  render() {
+    const { value, tooltip, onChange, required, i18n } = this.props;
+
+    return (
+      <FormGroup
+        label={
+          <Fragment>
+            {i18n._(t`Inventories`)}{' '}
+            {tooltip && (
+              <Tooltip position="right" content={tooltip}>
+                <QuestionCircleIcon />
+              </Tooltip>
+            )}
+          </Fragment>
+        }
+        fieldId="inventories-lookup"
+      >
+        <Lookup
+          id="inventories-lookup"
+          lookupHeader={i18n._(t`Inventories`)}
+          name="inventories"
+          value={value}
+          onLookupSave={onChange}
+          getItems={getInventories}
```

Comment:
What happens if we pass InventoriesAPI.read directly here?

Reply:
It would need to be bound with `.bind(InventoriesAPI)` -- but our lint rules want that put elsewhere outside the JSX, so I'm not sure it really saves much